### PR TITLE
auto_scan_test_unary_ops & Elu

### PR DESCRIPTION
**Pull request by qqj1130247885**
# Create A Good Pull Request

> 下面的文字请保留在PR说明的最后面，并在提完PR后，根据实际情况勾选确认以下情况  

Please check the follow step before merging this pull request

- [x] Python code style verification
- [x] Review all the code diff by yourself
- [ ] All models(TensorFLow/Caffe/ONNX/PyTorch) testing passed
- [ ] Details about your pull request, releated issues

If this PR add new model support, please update `model_zoo.md` and add model to out test model zoos(@wjj19950828)
- [ ] New Model Supported
- [ ] No New Model Supported
## 主要贡献
1 tests/onnx下 增加了针对一元输入，且输入对象为float32类型的OP的单测程序。
      包括：  "Cos", "Atan", "Asinh", "Asin","Acosh","Acos", "Cosh", "Exp","Floor", "Tan", "Erf","Sin",  "Sinh", "Tanh", "Atanh", "Sign"。
2 op_mapper增加了对若干OP 的map。
3 tests/onnx下,增加了对Elu的单测程序

### CODE REVIEW

File: tests/onnx/test_auto_scan_unarray_ops.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import hypothesis.strategies as st
+import onnx
+from onnx import helper
+from onnx import TensorProto
+import numpy as np
+import unittest
+import random
+
+min_opset_version_map = {
+    "IsInf": 10,
```
Comment (wjj19950828) [inline tests/onnx/test_auto_scan_unarray_ops.py]: IsInf去掉，非unary op，有具体属性变量，需要测到

File: tests/onnx/test_auto_scan_unarray_ops.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    "Elu": 7,
```
Comment (wjj19950828) [inline tests/onnx/test_auto_scan_unarray_ops.py]: Elu去掉，同理

File: tests/onnx/test_auto_scan_unarray_ops.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    "Asin": 7,
+    "Acosh": 9,
+    "Acos": 7,
+    "Exp": 7,
+    "Floor": 7,
+    "Tan": 7,
+}
+
+
+class TestUnarrayOpsConcert(OPConvertAutoScanTest):
```
Comment (wjj19950828) [inline tests/onnx/test_auto_scan_unarray_ops.py]: TestUnaryopsConvert

File: tests/onnx/test_auto_scan_unarray_ops.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    """
+    ONNX op: unarray ops
```
Comment (wjj19950828) [inline tests/onnx/test_auto_scan_unarray_ops.py]: unary

File: x2paddle/op_mapper/onnx2paddle/opset7.py [hunk 5]
```diff
@@ -32,6 +33,71 @@ def run_mapping(*args, **kwargs):
...
+            'Softplus': [
+                'paddle.nn.Softplus', dict(threshold='threshold'),
+                dict(threshold=float(sys.maxsize))
+            ],
+            'Exp': ['paddle.exp'],
+            'Log': ['paddle.log'],
+            'LogSoftmax': [
+                'paddle.nn.functional.log_softmax', dict(axis='axis'),
+                dict(axis=1)
+            ],
```
Comment (wjj19950828) [inline x2paddle/op_mapper/onnx2paddle/opset7.py]: 不需要把之前的拷贝过来，有继承关系，直接self.directly_map_ops.update即可

File: x2paddle/op_mapper/onnx2paddle/opset7.py [hunk 6]
```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 from .opset_legacy import OpSet
+import sys
```
Comment (wjj19950828) [inline x2paddle/op_mapper/onnx2paddle/opset7.py]: 去掉

**Review by wjj19950828 — CHANGES_REQUESTED** [review]

**Comment by wjj19950828** [comment]
文件名改一下，unary

File: x2paddle/op_mapper/onnx2paddle/opset7.py (hunk 6 above)
Comment (qqj1130247885) [reply x2paddle/op_mapper/onnx2paddle/opset7.py]: done

File: x2paddle/op_mapper/onnx2paddle/opset7.py (hunk 5 above)
Comment (qqj1130247885) [reply x2paddle/op_mapper/onnx2paddle/opset7.py]: done

File: tests/onnx/test_auto_scan_unarray_ops.py (hunk 4 above)
Comment (qqj1130247885) [reply tests/onnx/test_auto_scan_unarray_ops.py]: done

File: tests/onnx/test_auto_scan_unarray_ops.py (hunk 3 above)
Comment (qqj1130247885) [reply tests/onnx/test_auto_scan_unarray_ops.py]: done

File: tests/onnx/test_auto_scan_unarray_ops.py (hunk 2 above)
Comment (qqj1130247885) [reply tests/onnx/test_auto_scan_unarray_ops.py]: done

**Pull request closed without merging**